addons/website_event_meet/static/src/js/website_event_create_meeting_room_button.js

```diff
@@ -15,23 +15,19 @@ publicWidget.registry.websiteEventCreateMeetingRoom = publicWidget.Widget.extend
     //--------------------------------------------------------------------------
 
     _onClickCreate: async function () {
-        if (!this.$createModal) {
+        if (!this.createModal) {
             const langs = await rpc("/event/active_langs");
 
-            this.$createModal = $(renderToElement(
-                'event_meet_create_room_modal',
-                {
-                    csrf_token: odoo.csrf_token,
-                    eventId: this.$el.data("eventId"),
-                    defaultLangCode: this.$el.data("defaultLangCode"),
-                    langs: langs,
-                }
-            ));
-
-            this.$createModal.appendTo(this.$el.parentNode);
+            this.createModal = renderToElement("event_meet_create_room_modal", {
```

Comment:
This one: `createModelEl`: we render to an element

Reply:
@bso-odoo done!